tools/osx/INSTALL.readme.rtf

```diff
@@ -1,7 +1,7 @@
 To install the RawTherapee application:
 
 1. Open the RawTherapee .dmg Disk Image included alongside this document.
-2. Drag the RawTherapee icon onto the /Applications folder icon
+2. To install RT, drag icon onto /Applications (it will not start if in another location).
```

Comment:
"To install RT" is redundant due to the first line.

"RT" should be expanded to the full name. (I'm going by the RawPedia guidelines at https://rawpedia.rawtherapee.com/Contributing: "_Write "RawTherapee" instead of "RT" or "Rawtherapee". Never abbreviate "RawTherapee"._")

I think it can be more clear to the user that the installation location is a requirement, not a limitation. Instead of "it will not start if in another location", something to the effect of "RawTherapee MUST be installed in /Applications" would work. You can elaborate if you want, e.g. "For security reasons, RawTherapee MUST be installed in /Applications. RawTherapee will refuse to start if installed elsewhere.".

Reply:
Thanks @Lawrence37 for the tip. Updated PR per your suggestions.